src/System.Security.Principal/src/System.Security.Principal.csproj

```diff
@@ -22,8 +22,9 @@
     <Compile Include="System\Security\Principal\PrincipalPolicy.cs" />
     <Compile Include="System\Security\Principal\TokenImpersonationLevel.cs" />
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)' == 'net463'">
-    <TargetingPackReference Include="mscorlib" />
+  <ItemGroup>
+    <TargetingPackReference Include="mscorlib" Condition="'$(TargetGroup)' == 'net463'" />
+    <TargetingPackReference Include="System.Private.CoreLib" Condition="'$(TargetGroup)' != 'net463'" />
```

Comment:
This should not be needed - it should be building against System.Runtime like the current CoreCLR version.